app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,17 @@
+<nav class="osra-nav-header" role="navigation">
```

Comment:
Can we use Bootstrap's `navbar` classes here instead and avoid having to create and fine-tune our own?